### ISSUE TITLE
Project Board Migration: refactoring of "Issue Trigger" workflow

### ISSUE THREAD

**Issue by t-will-gillis** [hackforla/website#6996]
### Dependency
- [x] #7000 
- [x] #6995 

### Overview
Many of our workflows need to be refactored due to the forced migration from the Project Board (classic) to the Project Board- Beta. This issue addresses changes that we need to make in order to ensure that our "Issue Trigger" workflow works as it did previously.

### Action items
- [x] In [issue-trigger-yml](https://github.com/hackforla/website/blob/gh-pages/.github/workflows/issue-trigger.yml) apply the changes detailed in issue #7000 
- [x] In the same file, remove the entirety of the "Add-Issue-To-Project-Board" job including the two preceding comments: i.e. remove lines 7-21. The "Default workflows" will do this now.
- [x] In [preliminary-update-comment.js](https://github.com/hackforla/website/blob/gh-pages/github-actions/trigger-issue/add-preliminary-comment/preliminary-update-comment.js) there are some if statements that check columns and label assignments and post  appropriate messages (from #6811): comment out lines 54-87 +/-
- [x] Miscellaneous: add explicit semi-colons, consistent spacing, normalize comments
- [x] After making the changes, test the workflow in the new Projects. Create 'Draft' PR and stage for migration.

### Post migration issues needed:
- [ ] In [preliminary-update-comment.js](https://github.com/hackforla/website/blob/gh-pages/github-actions/trigger-issue/add-preliminary-comment/preliminary-update-comment.js) there are some if statements that will un-assign a member  and post a message if the member assigns themselves to an issue in the "New Issue Approval" column and the that issue has neither a `Complexity: Prework` nor `Draft` label. Since the new Projects does not include 'columns', the logic needs to be rewritten in terms of 'status' instead, using GraphQL. I.e. we need to redo PR #6811 
- [ ] Comment: for the issues with title "Hack for LA website bot..." from "inactive members", new issue previously was shuffled to "Questions / In Review" column >  
### Resources/Instructions
- [Sunset notice for classic Projects](https://github.blog/changelog/2024-05-23-sunset-notice-projects-classic/)
- This issue initiated by #6993

**Comment by ExperimentsInHonesty** [comment]
- [x] Need to move after pr actions to the epic

```
- [ ] Post migration issues needed after #6996 is closed
   - [ ] In [preliminary-update-comment.js](https://github.com/hackforla/website/blob/gh-pages/github-actions/trigger-issue/add-preliminary-comment/preliminary-update-comment.js) there are some if statements that will un-assign a member  and post a message if the member assigns themselves to an issue in the "New Issue Approval" column and the that issue has neither a `Complexity: Prework` nor `Draft` label. Since the new Projects does not include 'columns', the logic needs to be rewritten in terms of 'status' instead, using GraphQL. I.e. we need to redo PR #6811 
   - [ ] Comment: for the issues with title "Hack for LA website bot..." from "inactive members", new issue previously was shuffled to "Questions / In Review" column >  
### Resources/Instructions
- [Sunset notice for classic Projects](https://github.blog/changelog/2024-05-23-sunset-notice-projects-classic/)
- This issue initiated by #6993
